Add test for JSON object nesting depth (#784)
RedisJSON/RedisJSON: docs/docs/_index.md

Once the module has been loaded successfully, the Redis log should have lines similar to:

> ### Limitation

Reviewer: Should this limitation go after the log lines?

> A JSON value passed to a command can have a depth of up to 128. When passing to a command a JSON value containing an object or an array with a nesting level of more than 128, then the command will return an error.

Reviewer: ```suggestion
A JSON value passed to a command can have a depth of up to 128. If you pass to a command a JSON value that contains an object or an array with a nesting level of more than 128, the command returns an error.
```
